content/ADR165-resolving-component-numbers.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: adr
+adr: 165 # replace this number for the PR or ISSUE number
+title: Resolving component numbers for the scripting of the SDK
+date: 2023-01-04
+status: Draft
+type: RFC # pick one of these
+spdx-license: CC0-1.0
+authors:
+  - menduz
+---
+
+## Abstract
+
+This RFC describes the non-functional problems to solve in relation to the component numbers of the SDK. It presents synchronization, usability and ergonomic problems behind the technical requirements for the performance. It presents alternatives as a jurney, to add context on the decisions. The document is still not conclusive.
```

Comment:
```suggestion
This RFC describes the non-functional problems to solve in relation to the component numbers of the SDK. It presents synchronization, usability and ergonomic problems behind the technical requirements for the performance. It presents alternatives as a journey, to add context on the decisions. The document is still not conclusive.
```

Reply:
I would suggest this for more variety in the used words:
```suggestion
This RFC describes the non-functional issues to solve about the component numbers of the SDK. It presents synchronization, usability, and ergonomic problems behind the technical requirements for the performance. It extends alternatives as a journey to add context to the decisions. The document is still not conclusive.
```